WORKSPACE

```diff
@@ -27,25 +27,25 @@ dpkg_src(
     name = "debian_stretch",
     arch = "amd64",
     distro = "stretch",
-    sha256 = "2b13362808b7bd90d24db2e0804c799288694ae44bd7e3d123becc191451fc67",
-    snapshot = "20191230T150135Z",
+    sha256 = "56537cedf58e6f08bb3eafef514a20016fbfd227850ab810c43e5ffb00f57427",
+    snapshot = "20200219T150721Z",
```

Comment:
If this value is always the same everywhere can we just put it in a variable at the top? Or is that not bazel-y?

Reply:
See the other comment.

Reply:
I just go to (for example) https://snapshot.debian.org/archive/debian-security/ and select the most recent timestamp.